### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by JakeSCahill**

### CODE REVIEW

File: docs/modules/ROOT/pages/get-started.adoc [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+:description: In this tutorial, you'll deploy a Hazelcast cluster and an instance of Management Center, using Hazelcast Platform Operator for Kubernetes.
 
-The simplest way of installing and operating your Hazelcast Platform cluster in the Kubernetes (and OpenShift) environments is to use Hazelcast Platform Operator.
+{description}
 
-== Deploy Hazelcast Platform Operator
+== Step 1. Deploy Hazelcast Platform Operator
 
-You can deploy Hazelcast Platform Operator by applying the https://repository.hazelcast.com/operator/bundle.yaml[`bundle.yaml`] file. Please apply it to your Kubernetes cluster with the following command.
+To deploy Hazelcast Platform Operator, use the following command to apply the https://repository.hazelcast.com/operator/bundle.yaml[`bundle.yaml`] file to your Kubernetes cluster:
```
Comment (alparslanavci) [inline docs/modules/ROOT/pages/get-started.adoc:8]: The bundle.yaml link should be as follows:

```
https://repository.hazelcast.com/operator/bundle-latest-snapshot.yaml
```

File: docs/modules/ROOT/pages/get-started.adoc [hunk 2]
```diff
@@ -225,3 +242,7 @@ Finally, run the command below to delete Hazelcast Platform Operator deployment.
 ----
 kubectl delete -f https://repository.hazelcast.com/operator/bundle.yaml
```
Comment (alparslanavci) [inline docs/modules/ROOT/pages/get-started.adoc:243]: Same here, we need to update bundle.yaml link.

**Review by alparslanavci — CHANGES_REQUESTED** [review]
@JakeSCahill, thanks for the PR, could you please also update the bundle.yaml links?

**Comment by JakeSCahill** [comment]
Sure 😄  For 5.0, will it be bundle-5.0.yaml? Or bundle-latest.yaml?

**Comment by alparslanavci** [comment]
Thanks!

For 5.0, it should be `bundle-5.0.yaml`. For the `latest`, it should be `bundle-latest.yaml` :)

**Comment by JakeSCahill** [comment]
So for now, `bundle-5.0` and `bundle-latest` will point to the same file?

Will `latest` always have a versioned counterpart?

**Comment by alparslanavci** [comment]
Exactly 👍

**Comment by JakeSCahill** [comment]
Cool, I've updated the path to use the `full-version` attribute. This will be derived from the `antora.yml` file of each docs version.

**Review by alparslanavci — APPROVED** [review]
Thanks @JakeSCahill!

**Pull request merged**